### PR TITLE
[14.0] [MIG] account_check_printing_report_base

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

**Review by hildickethan — APPROVED** [review]
Functional review, but I would recommend changing the last 2 commits to fit the migration standard of `[IMP] account_check_printing_report_base: black, isort, prettier` and `[MIG] account_check_printing_report_base: Migration to 14.0`

**Comment by kittiu** [comment]
@AmmarOfficewalaSerpentCS are you still working on this PR? Can you attend the comment of @hildickethan please.

**Comment by AmmarOfficewalaSerpentCS** [comment]
@kittiu ,
I am working on a PR

**Comment by kittiu** [comment]
> @kittiu ,
> I am working on a PR

You mean, you will continue with this PR, right? :)

File: account_check_printing_report_base/report/account_check_writing_report.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<odoo>
+    <report
```
Comment (kittiu) [inline account_check_printing_report_base/report/account_check_writing_report.xml:2]: This tag is deprecated in Odoo 14. Please use <record> instead.

**Comment by etobella** [comment]
@AmmarOfficewalaSerpentCS Any news?

**Comment by litmount** [comment]
> @AmmarOfficewalaSerpentCS Any news?

I have the same question. Is this still being worked on @AmmarOfficewalaSerpentCS ?

**Comment by AmmarOfficewalaSerpentCS** [comment]
@litmount ,
I will fix this issue in a 15 days

**Comment by JuanyDForgeflow** [comment]
@AmmarOfficewalaSerpentCS Do you plan to fix this?

**Comment by AmmarOfficewalaSerpentCS** [comment]
@JuanyDForgeflow , 
Yes, I fixed everything which commented

**Comment by JuanyDForgeflow** [comment]
@AmmarOfficewalaSerpentCS seems like you have to run precommit again and it'll be ready

**Comment by AmmarOfficewalaSerpentCS** [comment]
@JuanyDForgeflow , 

Now Everything is Green

**Review by cesar-tecnativa — APPROVED** [review]

File: account_check_printing_report_base/report/account_check_writing_report.xml [hunk 2]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" ?>
```
Comment (pedrobaeza) [inline account_check_printing_report_base/report/account_check_writing_report.xml]: Why removing this? Better to put complete form

File: account_check_printing_report_base/report/check_print.py [hunk 3]
```diff
@@ -108,9 +117,10 @@ def _get_report_values(self, docids, data=None):
         model = self.env.context.get("active_model", "account.payment")
         objects = self.env[model].browse(docids)
         paid_lines = self.get_paid_lines(objects)
+
```
Comment (pedrobaeza) [inline account_check_printing_report_base/report/check_print.py:120]: Don't add empty lines in the middle of the method

File: account_check_printing_report_base/report/lang.py [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
 from num2words import CONVERTER_CLASSES, CONVERTES_TYPES, num2words
-from num2words.base import Num2Word_Base
 from num2words.lang_ES import Num2Word_ES
 
 num2words_by_lang = {"es": "Num2WordESCustom"}
 
 
 class Num2WordESCustom(Num2Word_ES):
-    CURRENCY_FORMS = Num2Word_ES.CURRENCY_FORMS.copy()
-    CURRENCY_FORMS.update({"EUR": (("euro", "euros"), ("céntimo", "céntimos"))})
```
Comment (pedrobaeza) [inline account_check_printing_report_base/report/lang.py]: Why removing this?

File: account_check_printing_report_base/tests/test_check_printing_report.py [hunk 5]
```diff
@@ -12,11 +12,6 @@
 class TestAccountCheckPrintingReportBase(TransactionCase):
     def setUp(self):
         super().setUp()
-        self.langs = ("en_US", "es_ES")
```
Comment (pedrobaeza) [inline account_check_printing_report_base/tests/test_check_printing_report.py]: Please don't remove this, as it's needed for checking conditions on Spanish

File: account_check_printing_report_base/tests/test_check_printing_report.py [hunk 6]
```diff
@@ -70,27 +65,6 @@ def setUp(self):
                 ],
             }
         )
-        self.acc_payable = self._create_account(
```
Comment (pedrobaeza) [inline account_check_printing_report_base/tests/test_check_printing_report.py]: Don't remove this. It's for being independent from the installed CoA.

File: account_check_printing_report_base/tests/test_check_printing_report.py [hunk 7]
```diff
@@ -137,65 +112,77 @@ def _create_invoice_line(self, account, invoice):
 
         return invoice
 
-    def test_01_check_printing_no_layout(self):
```
Comment (pedrobaeza) [inline account_check_printing_report_base/tests/test_check_printing_report.py]: Why are you removing some tests?

File: account_check_printing_report_base/tests/test_check_printing_report.py [hunk 8]
```diff
@@ -137,65 +112,77 @@ def _create_invoice_line(self, account, invoice):
...
         amount1 = self.report.fill_stars_number(str(amount))
         amount2 = self.report.fill_stars(amount_in_word)
-        self.assertEquals(amount1, "***** %s *" % amount)
-        self.assertEquals(amount2, "{} {}".format(amount_in_word, ("*" * stars)))
+        amount3 = self.report.fill_stars(large_amount_in_word)
+        self.assertEqual(amount1, "***** %s *" % amount)
+        self.assertEqual(amount2, "{} {}".format(amount_in_word, ("*" * stars)))
+        self.assertEqual(amount3, large_amount_in_word)
 
-    def test_num2words(self):
```
Comment (pedrobaeza) [inline account_check_printing_report_base/tests/test_check_printing_report.py]: Don't remove tests

File: account_check_printing_report_base/views/account_journal_view.xml [hunk 9]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8" ?>
```
Comment (pedrobaeza) [inline account_check_printing_report_base/views/account_journal_view.xml]: Don't remove these headers

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Pull request closed without merging**